### PR TITLE
Fix issue that width of Dashboard is somehow fixed. Issue: #29

### PR DESCRIPTION
**Pull request by yizheliu-amazon**
*Issue #, if available:*
https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/29
*Description of changes:*
Fix issue that width of Dashboard is somehow fixed, which causes right-side components will get hidden or partially hidden.

After fix, even if the window shrinks, the button is still visible. 
Screen Shot 2020-04-16 at 6 26 11 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/pages/main/Main.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ export function Main(props: MainProps) {
       <EuiPageSideBar style={{ minWidth: 150 }} hidden={hideSideNavBar}>
         <EuiSideNav style={{ width: 150 }} items={sideNav} />
       </EuiPageSideBar>
-      <EuiPageBody>
+      <EuiPageBody restrictWidth={'70%'}>
```
Comment (ylwu-amzn) [inline public/pages/main/Main.tsx:73]: How about we remove this restriction to get wider page, especially for detector list

Comment (yizheliu-amazon) [reply public/pages/main/Main.tsx:73]: Good point. It turns out that the min width of bottom 2 components `sunburst chart` + `detector feature list` is around 800px. And min width of Side Nav Bar is 150px. The smallest width which has no hidden contents is around 1000px.

Screenshot shows the above smallest width on mac 13in screen. 

Screen Shot 2020-04-16 at 9 16 39 PM

Comment (yizheliu-amazon) [reply public/pages/main/Main.tsx:73]: Is that smallest width acceptable?

Comment (ylwu-amzn) [reply public/pages/main/Main.tsx]: I think that's ok considering current popular resolution is above 1000px

File: public/pages/Dashboard/index.scss [hunk 2]
```diff
@@ -24,13 +24,10 @@
...
-  height: 28px;
-  width: 505px;
   color: #879196;
   font-family: 'Helvetica Neue';
   font-size: 12px;
   letter-spacing: 0;
-  line-height: 14px;
 }
 
 .live-badge {
```
Comment (ylwu-amzn) [inline public/pages/Dashboard/index.scss:33]: Do we still need this ?

Comment (yizheliu-amazon) [reply public/pages/Dashboard/index.scss:33]: no. Good catch. will remove it.

**Pull request merged**